mtools/mloginfo/sections/connection_section.py

```diff
@@ -19,6 +20,7 @@ def __init__(self, mloginfo):
 
         # add --restarts flag to argparser
         self.mloginfo.argparser_sectiongroup.add_argument('--connections', action='store_true', help='outputs information about opened and closed connections')
+        self.mloginfo.argparser_sectiongroup.add_argument('--stats', action='store_true', help='outputs helpful statistics for connection duration (min/max/avg)')
```

Comment:
I think you can make this more intuitive to use:
  -`--stats` is too generic given this option is specific to connections.
 - The `--stats` option doesn't do anything unless you also specify `--connections`.

Suggest you make a top level connection-stats section so the command line usage would be `mloginfo --connection-stats ...`.

Reply:
done now `mloginfo --connection --connstats logfile` and `mloginfo --connstats logfile` have same meaning